api/views.py

```diff
@@ -561,7 +573,7 @@ class UserView(APIView):
 
     def get(self, request, format=None):
```

Comment:
This GET to UserView has been deleted in main (it's the old way to generate users before we were using PGP). I guess this is the cause of the current merge conflict